hare/eligibility/beacon.go

```diff
@@ -34,5 +37,23 @@ func (b *beacon) Value(layer types.LayerID) (uint32, error) {
 		return nilVal, err
 	}
 
-	return v, nil
+	return calcValue(v), nil
+}
+
+// calculates the beacon value from the set of ids
+func calcValue(bids map[types.BlockID]struct{}) uint32 {
```

Comment:
Where was this code before this PR? How did we calculate the beacon before?

Reply:
tortoise uses the same code, but the tortoise just needs it to use as map keys (cant use slices) and i think 32 might be enough there, if you want to change it id rather do it in a different task

Reply:
I agree